tests/spread/integration/aspnetcore-runtime-9.0/task.yaml

```diff
@@ -0,0 +1,25 @@
+summary: Integration tests for ASP.NET 9 Runtime
+
+execute: |
+  # install slices
+  rootfs="$(install-slices aspnetcore-runtime-9.0_libs base-files_base)"
+  mv "${rootfs}"/usr/lib/dotnet/dotnet9 "${rootfs}"/usr/lib/dotnet/dotnet
+  ln -s ../lib/dotnet/dotnet "${rootfs}"/usr/bin/dotnet
+
+  # smoking test the .NET runtime
```

Comment:
We can remove this test since we already run it in `tests/spread/integration/dotnet-host-9.0/task.yaml`.